scripts/rebuild-ui.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euxf -o pipefail
+
+cd jaeger-ui
+
+tag=$(git describe --exact-match --tags)
+if [[ "$tag" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]];  then
+    temp_file=$(mktemp)
+    trap "rm -f ${temp_file}" EXIT
+    release_url="https://github.com/jaegertracing/jaeger-ui/releases/download/${tag}/assets.tar.gz"
+
+     if curl -O "$release_url"; then
```

Comment:
when I ran this, I got `assets.tar.gz` in the local dir. Why do you need to call curl twice?

Reply:
I guess it was part of some earlier commit wherein I was first checking if the file exists instead of downloading right away. I have fixed this.